src/league-of-legends-api/poll-current-game.ts

```diff
@@ -107,15 +109,57 @@ export const pollCurrentGame = (
             }
             break;
           }
-          // case LoLClientEvent.ACE: {
-          //   console.log('ace occured!');
-          //   await playClip(
-          //     `${pathToClips}halo_unfreakinbelievable.mp3`,
-          //     channel,
-          //     audioPlayer
-          //   );
-          //   break;
-          // }
+          case LoLClientEvent.ACE: {
+            console.log('ace occured!');
+            await playClip(`${pathToClips}halo_unfreakinbelievable.mp3`, channel, audioPlayer);
+            break;
+          }
+          case LoLClientEvent.MINIONS_SPAWNING: {
+            console.log('minions spawning...');
+            await playClip(`${pathToClips}minions_laugh.mp3`, channel, audioPlayer);
+            break;
+          }
+          case LoLClientEvent.FIRST_TOWER: {
+            console.log('first turret destroyed!');
+            break;
+          }
+          case LoLClientEvent.TURRET_KILLED: {
+            console.log('turret killed!');
+            break;
+          }
+          case LoLClientEvent.INHIB_KILLED: {
+            console.log('inhib killed!');
+            break;
+          }
+          case LoLClientEvent.INHIB_RESPAWNED: {
+            console.log('inhib respawned...');
+            break;
+          }
+          case LoLClientEvent.DRAGON_KILLED: {
+            if (newEvent.Stolen === 'True') {
+              console.log('dragon stolen!');
+              await playClip(`${pathToClips}steal_kims_convenience.mp3`, channel, audioPlayer);
+            } else {
+              console.log('dragon killed!');
+              await playClip(`${pathToClips}dracarys.mp3`, channel, audioPlayer);
+            }
+            break;
+          }
+          case LoLClientEvent.HERALD_KILLED: {
+            if (newEvent.Stolen === 'True') {
+              console.log('herald stolen!');
+              await playClip(`${pathToClips}steal_kims_convenience.mp3`, channel, audioPlayer);
+            }
+            console.log('herald killed!');
+            break;
+          }
+          case LoLClientEvent.BARON_KILLED: {
+            if (newEvent.Stolen === 'True') {
+              console.log('baron stolen!');
+              await playClip(`${pathToClips}steal_kims_convenience.mp3`, channel, audioPlayer);
+            }
+            break;
+          }
```

Comment:
Unrelated work... I didn't want to make a 3rd branch of a branch. It's a separate commit I don't intend to squash